packages/backend/src/persistance/storage/postgres/getNumberOfMessages.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client';
+
+export const getNumberOfMessages =
+    (db: PrismaClient) => async (ensName: string, contactName: string) => {
+        const account = await db.account.findFirst({
+            where: {
+                id: ensName,
+            },
+        });
+        if (!account) {
+            return 0;
+        }
+        const conversation = await db.conversation.findFirst({
```

Comment:
A conversation is the (encrypted)name of the contact the user has a chat with. Contacts would be the better name to describe it but conversation is already widely used in both other parts of the app and the specification aswell. So I think it would make sense to go with "conversation" instead of "contact"